docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -65,32 +74,56 @@ Use these controls to filter what's included in the logs table:
 
 [float]
-[[manual-runs-table]]
-==== Manual runs table
+[[gaps-table]]
+==== Gaps table
 
 beta::[]
 
-Each manual run can produce multiple rule executions, depending on the time range of the run and the rule's execution schedule. These details are shown in the Manual runs table.
+Gaps in rule executions are periods of time where a rule didn’t run as expected. These gaps can be caused by various disruptions, including system updates, rule failures, or simply turning off a rule. Addressing them is essential for maintaining consistent coverage and avoiding missed alerts. 
 
-To access the table, navigate to the detection rules page, click the rule's name to open its details, then scroll down and select the **Execution results** tab. Scroll down again to find the Manual runs table. 
+TIP: Refer to the <<troubleshoot-gaps>> section for strategies for avoiding gaps.
 
-To stop an active run, go to the appropriate row and click **Stop run** in the **Actions** column. Completed rule executions for each manual run are logged in the Execution log table.
+Use the information in the Gaps table to assess the scope and severity of rule execution gaps. To control what's shown in the table, you can filter the table by gap status, select a time range for viewing gap data, and sort multiple columns. 
 
 [role="screenshot"]
-image::images/manual-rule-run-table.png[Manual rule runs table on the rule execution results tab]
+image::images/gaps-table.png[Gaps table on the rule execution results tab]
 
-The Manual runs table displays important details such as:
+The Gaps table has the following columns:
 
-* The status of each manual run:
-** **Pending**: The rule is not yet running. 
-** **Running**: The rule is executing during the time range you specified. Some rules, such as indicator match rules, can take longer to run.
-** **Error**: The rule's configuration is preventing it from running correctly. For example, the rule's conditions cannot be validated.
+* **Status**: The current state of the gap. It can be `Filled`, `Partially filled`, or `Unfilled`.
+* **Detected at**: The date and time the gap was first discovered.
+* **Manual fill tasks**: The status of the manual run that’s filling the gap. For more details about the manual run, refer to its entry in the <<manual-runs-table,manual run table>>.
+* **Event time covered**: How much progress the manual run has made filling the gap. 
++
+NOTE: If you stop a manual run that's filling a gap, the gap’s status is set to `Partially filled`. To fill the remaining gap, you can select the **Fill remaining gap** action or <<manually-run-rules,manually run>> the rule over the gap's time frame.
```

Comment:
> If you stop a manual run that's filling a gap, the gap’s status is set to `Partially filled`


It really depends on what this task was able to do before user canceled it:

- if task was scheduled, but not yet execute any rule - gap will remain unfilled
- in case if there manual rule run larger than gap range, and user cancel it - gap can be filled if all execution related to gap was done

Reply:
How's this:
 
```suggestion
NOTE: If you stop a manual run that hasn't finished filling a gap, the gap’s status will be set to `Partially filled`. To fill the remaining gap, you can select the **Fill remaining gap** action or <<manually-run-rules,manually run>> the rule over the gap's time frame.
```